app/assets/stylesheets/application.scss

```diff
@@ -52,6 +56,22 @@ p.text {
   color: #D5CAC8;
 }
 
+.text {
+  color: #ffffff;
+}
+
+.green {
+  color: #397D02;
+}
+
+.white {
+  color: #D5CAC8;
+}
+
+.blue {
+  color: #23527C;
+}
+
 .navbar {
   background: url("http://subtlepatterns2015.subtlepatterns.netdna-cdn.com/patterns/outlets.png");
```

Comment:
Consider hosting images like this under your app rather than being dependent on a 3rd party for this. They could change the URL or their site goes down and it'd affect your site. By hosting this image within the app its also available to you if you happen to work offline.